lua/terrortown/entities/roles/ttt_role_base/shared.lua

```diff
@@ -273,3 +273,13 @@ function ROLE:CanUseTraitorButton()
 
 	return cv and cv:GetBool() or false
 end
+
+---
+-- Returns whether a @{ROLE} can see traitor buttons.
+-- @return boolean
+-- @realm shared
+function ROLE:CanSeeTraitorButton()
+	local cv = GetConVar("ttt_" .. self.name .. "_traitor_button_visible")
+
+	return cv and cv:GetBool() or false
+end
```

Comment:
It might be better to check if the player can use the buttons here and return true if they can.

This would be more clear and would make using this in the render code easier, since we can eliminate the access or canSee stuff with just this one function call.

We might also need to add a note to the documentation, saying that this is ignored once a role is able to use traitor buttons.

Reply:
ok, set it to call CanUse*() first